.github/actions/import-data/src/importer.ts

```diff
@@ -86,10 +86,10 @@ const createPolicy = async (accountId: number, pack: string) => {
 	return response.alertsPolicyCreate.id;
 };
 
-const createDashboardLocal = async (accountId: number, pack: string) => {
+const createDashboardLocal = async (accountId: number, quickstart: string) => {
 	const replacer = new RegExp('"accountId":0', 'g');
 
-	const dir = `${__dirname}/../../../../packs/${pack}/dashboards`;
+	const dir = `${__dirname}/../../../../quickstarts/${quickstart}/dashboards`;
```

Comment:
One thing that stands out here - quickstarts can now be nested, so this likely doesn't work unless the user realizes `path/to/quickstart` is how the input would need to be structured

Reply:
ya good point, i think we can follow up on this post GA